Session 2

HiiYL opened the pull request.

Commit:
Added POST example

Commit:
Added GET Example

Commit:
Added random images

Commit:
Session 2 stuff

Commit:
Corrected indentation separated fiels into partials and added basic search

Commit:
Added all isset checks

anonoz commented on _home.php:
```diff
@@ -0,0 +1,57 @@
+<?php
+  $phone_numbers = array (
```
@HiiYL `array(` not `array (` would be better

anonoz commented:
Lemme fix this PR before merging OK

anonoz commented:
@hackerspacemmu Can we change home.php to index.php?

anonoz committed:
- Beautified some codes
- Removed PHP5.6 shorthand array syntax
- Softened tonality of ternary operator comment
